src/Resources/contao/library/Contao/Model.php

```diff
@@ -1083,7 +1083,7 @@ protected static function find(array $arrOptions)
 
 		if ($objResult->numRows < 1)
 		{
-			return null;
+			return $arrOptions['return'] == 'Array' ? [] : null;
```

Comment:
Since this is "old" code, you need to replace `[]` with `array()`. Then its `RTM` IMO.